airbyte-integrations/connectors/source-aws-cloudtrail/source_aws_cloudtrail/spec.json

```diff
@@ -14,20 +14,24 @@
     "properties": {
       "aws_key_id": {
         "type": "string",
-        "description": "Specifies an AWS access key associated with an IAM user or role.",
```

Comment:
Same as described in this comment: https://github.com/airbytehq/airbyte/pull/8434/files#r763024940